falcon/routing/converters.py

```diff
@@ -59,13 +60,16 @@ class IntConverter(BaseConverter):
     __slots__ = ('_num_digits', '_min', '_max')
 
     def __init__(self, num_digits=None, min=None, max=None):
-        if num_digits is not None and num_digits < 1:
-            raise ValueError('num_digits must be at least 1')
 
+        self.validateNumberOfDigits(num_digits)
         self._num_digits = num_digits
         self._min = min
         self._max = max
 
+    def validateNumberOfDigits(self, num_digits):
```

Comment:
I'd suggest using `snake_case` instead of `camelCase`, in order to follow the best Python practices and recommendations.

See: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names